lib/perl/Genome/Model/ReferenceAlignment/Command/RefCovMetrics.pm

```diff
@@ -0,0 +1,124 @@
+package Genome::Model::ReferenceAlignment::Command::RefCovMetrics;
+
+use strict;
+use warnings 'FATAL';
+
+use Genome;
+
+require List::MoreUtils;
+
+class Genome::Model::ReferenceAlignment::Command::RefCovMetrics {
+    is => 'Command::V2',
+    has_input => {
+        models => {
+            is => 'Genome::Model::ReferenceAlignment',
```

Comment:
@tmooney  That is a possibility. To do so:
* move the command to a different namespace (genome instdata alignment-result merged?)
* pick a different row identifier for each result (it is currently model_name)

Additionally we could allow different 'get' methods: instdata, models, result id). What do yo think?